wsmaster/codenvy-hosted-api-organization/src/main/java/com/codenvy/organization/api/resource/OrganizationResourcesDistributionService.java

```diff
@@ -0,0 +1,166 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.organization.api.resource;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+import com.codenvy.organization.api.DtoConverter;
+import com.codenvy.organization.shared.dto.OrganizationDistributedResourcesDto;
+import com.codenvy.organization.shared.model.OrganizationDistributedResources;
+import com.codenvy.resource.api.free.ResourceValidator;
+import com.codenvy.resource.model.Resource;
+import com.codenvy.resource.shared.dto.ResourceDto;
+
+import org.eclipse.che.api.core.BadRequestException;
+import org.eclipse.che.api.core.ConflictException;
+import org.eclipse.che.api.core.NotFoundException;
+import org.eclipse.che.api.core.Page;
+import org.eclipse.che.api.core.ServerException;
+import org.eclipse.che.api.core.rest.Service;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static java.lang.String.format;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+/**
+ * REST API for resources distribution between suborganizations
+ *
+ * @author Sergii Leschenko
+ */
+@Api(value = "/organization/resource", description = "REST API for resources distribution between suborganizations")
+@Path("/organization/resource")
+public class OrganizationResourcesDistributionService extends Service {
+    private final OrganizationResourcesDistributor resourcesDistributor;
+    private final ResourceValidator                resourceValidator;
+
+    @Inject
+    public OrganizationResourcesDistributionService(OrganizationResourcesDistributor resourcesDistributor,
+                                                    ResourceValidator resourceValidator) {
+        this.resourcesDistributor = resourcesDistributor;
+        this.resourceValidator = resourceValidator;
+    }
+
+    @POST
+    @Path("{suborganizationId}")
```

Comment:
`reset` method path starts with `/` while this path doesn't, i find it a bit inconsistent don't you?